assignments/objects.js

```diff
@@ -49,16 +107,28 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {'name': 'Susan', 'age': 70,  speak: function() {
```

Comment:
Try nesting the "child" object as key of the parent object, then do the same for grandchild to child